src/Tile/_ProductTileMedia.js

```diff
@@ -1,28 +1,20 @@
-import classnames from 'classnames';
 import PropTypes from 'prop-types';
 import React from 'react';
+import TileMedia from './_TileMedia';
 
 const ProductTileMedia = props => {
-    const { image, className, ...rest } = props;
-
-    const tileProductMediaClasses = classnames(
-        'fd-product-tile__media',
-        className
-    );
+    const { image } = props;
 
     return (
-        <div
-            {...rest}
-            className={tileProductMediaClasses}
+        <TileMedia productTile {...props}
             style={{ backgroundImage: 'url(' + image + ')' }} />
     );
 };
 
 ProductTileMedia.displayName = 'ProductTile.Media';
 
 ProductTileMedia.propTypes = {
-    image: PropTypes.string.isRequired,
-    className: PropTypes.string
+    image: PropTypes.string.isRequired
```

Comment:
In the `propDescriptions` for this prop, it should probably say "URL of the background image" so it's clearer to the consumer what is to be expected.